docs/source/getting-started/tabular-data/index.rst

```diff
@@ -86,18 +85,30 @@ The compositor is the foundation of our differentially private queries. It essen
            (): dp.Margin(max_partition_length=60_000_000),
        },
    )
+   # Once you construct the context, you should abstain from directly accessing your data again.
+   # In fact, it is good practice to delete it! 
+   del df
```

Comment:
(github won't let me suggest the whole block; sorry.)
```suggestion
.. init code that we don't want cluttering the example:
    >>> import polars as pl
    >>> df = pl.LazyFrame()

.. code-block:: python

    >>> context = dp.Context.compositor(
    ...     data=df,
    ...     privacy_unit=dp.unit_of(contributions=36),
    ...     privacy_loss=dp.loss_of(epsilon=1.0),
    ...     split_evenly_over=10,
    ...     margins={
    ...         ("YEAR", ): dp.Margin(max_partition_length=60_000_000, max_partition_contributions=4),
    ...         ("YEAR", "QUARTER",): dp.Margin(max_partition_length=60_000_000, max_partition_contributions=1),
    ...         (): dp.Margin(max_partition_length=60_000_000),
    ...     },
    ... )
    
    >>> # Once you construct the context, you should abstain from directly accessing your data again.
    >>> # In fact, it is good practice to delete it! 
    >>> del df
```

Also: In the notebook conversions, I've been moving code into tabs, even when we only have Python right now. For consistent style, would we like to say that it should also be wrapped in a tab? Or maybe that's gratuitious?

Reply:
I'm fine with tabs, I expect we'll be adding more as we go.